books/free-programming-books-subjects.md

```diff
@@ -480,6 +480,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Bayesian Methods for Hackers](https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers) - Cameron Davidson-Pilon
 * [Beginning and Intermediate Algebra](http://www.wallace.ccfaculty.org/book/book.html) - Tyler Wallace
 * [Book of Proof](https://www.people.vcu.edu/~rhammack/BookOfProof/) - Richard Hammack [(PDF)](https://www.people.vcu.edu/~rhammack/BookOfProof/Main.pdf)
+* [Calculus](https://ocw.mit.edu/courses/res-18-001-calculus-fall-2023/) - Gilbert Strang (PDF)
```

Comment:
shouldn't we link to the textbook bage, https://ocw.mit.edu/courses/res-18-001-calculus-fall-2023/pages/textbook/ ?

Reply:
Now linking directly to the textbook page.